tutorials/deploy-instances-packer-terraform/index.mdx

```diff
@@ -58,11 +58,11 @@ Packer uses JSON files for the configuration.
 
 The following are the **required** configuration parameters for a minimal configuration file:
 
-- `project_id` (string) - The project ID to use to identify your project. It can also be specified via environment variable `SCW_DEFAULT_PROJECT_ID`.
+- `project_id` (string) - The project ID to use to identify your project. It can also be specified via the environment variable `SCW_DEFAULT_PROJECT_ID`.
```

Comment:
```suggestion
- `project_id` (string) - The project ID to use to identify your Project. It can also be specified via the environment variable `SCW_DEFAULT_PROJECT_ID`.
```